server/data/repositories/user.repository.js

```diff
@@ -58,6 +61,17 @@ class UserRepository extends BaseRepository {
       }
     });
   }
+
+  findUserByLetter(username) {
+    return this.model.findAll({
+      where: {
+        username: {
+          [Op.startsWith]: username
```

Comment:
Will this search handle both uppercase and lowercase? Our usernames are case sensitive, Op.startsWith is also case sensitive. It depends on the requirements. As a user, I would expect that search would return both lowercase and uppercase versions.
https://stackoverflow.com/questions/42352090/sequelize-find-all-that-match-contains-case-insensitive Here's a suggestion how to search regardless of case. Not sure if we need to implement it though, but could be useful.

Reply:
This is really important, thanks. 
Fixed